net/mwan3/files/etc/init.d/mwan3

```diff
@@ -41,6 +41,12 @@ start_service() {
 	mwan3_set_policies_iptables
 	mwan3_set_user_rules
 
+	procd_open_instance
+	procd_set_param command /usr/sbin/mwan3d
```

Comment:
I think we want to add `procd_set_param respawn` here

Reply:
We can add some logs and state cleanups if needed, but we need to be as resilient as posible, and needing to connect and restart a service is definitely not acceptable.

Reply:
I would prefer if not only this but all processes of the service would be restarted. But you are right, enabling this is better than not enabling it.